app/src/main/AndroidManifest.xml

```diff
@@ -20,21 +19,23 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".feature.atm_consultancy.AtmConsultancyActivity"
             android:label="@string/atm_consultancy"
             android:theme="@style/Theme.AtmConsultancy" />
-
         <activity
             android:name=".feature.media_player.activities.VideoPlayerActivity"
             android:screenOrientation="landscape"
             android:theme="@style/Theme.AndroidUdemyCourse.NoActionBar" />
-
         <activity
             android:name=".feature.media_player.activities.MediaPlayerActivity"
             android:label="@string/media_player"
             android:theme="@style/Theme.AndroidUdemyCourse.NoActionBar" />
+        <activity
```

Comment:
o learn english ele esta utilizando activity ou apenas fragments? se for apenas fragments pode remover esse trecho de codigo